alma/views/templates/admin/_configure/helpers/form/15/form.tpl

```diff
@@ -285,21 +285,21 @@
                                                 {if $key == 'months'}
                                                     {*
                                                         This comment is useful to the translator tools /!\ do not remove them
-                                                        {l s='January'}
-                                                        {l s='February'}
-                                                        {l s='March'}
-                                                        {l s='April'}
-                                                        {l s='May'}
-                                                        {l s='June'}
-                                                        {l s='July'}
-                                                        {l s='August'}
-                                                        {l s='September'}
-                                                        {l s='October'}
-                                                        {l s='November'}
-                                                        {l s='December'}
+                                                        {l s='January' mod='alma'}
```

Comment:
I don't like at all... but I can't see how to make it differently 😞

Reply:
@syjust-alma 
This is the native code of Prestashop 1.5. But when we redo the module configuration, if we can try to remove this overload it would be nice.🙏